dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -60,6 +60,17 @@ LABEL \
 ENV ELASTIC_CONTAINER "true"
 ENV PATH={{ $beatHome }}:$PATH
 
+# Add an init process, check the checksum to make sure it's a match
+RUN set -e ; \
+  TINI_VERSION='v0.19.0' ; \
+  TINI_BIN='tini-amd64' ; \
```

Comment:
There is also a `tini-static-amd64` binary, should we use it to avoid possible problems with dependencies? (though it only depends on libc)

Reply:
Not needed, let's ignore this by now.